web-services/query/src/main/java/datawave/webservice/query/runner/BasicQueryBean.java

```diff
@@ -100,6 +102,10 @@ public class BasicQueryBean {
     @Resource
     private SessionContext sessionContext;
     
+    @Inject
+    @SpringBean(name = "ResponseObjectFactory")
```

Comment:
Do we need `@SpringBean` here or can we follow the pattern established throughout this PR where we just use an `@Inject`?